engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/NodeDependency.scala

```diff
@@ -59,3 +60,7 @@ object DualParameterEditor {
     Decoder.decodeString.emapTry(name => Try(DualEditorMode.fromName(name)))
   }
 }
+
+@ConfiguredJsonCodec sealed trait ParameterValidator
+
+case object NotEmptyValidator extends ParameterValidator
```

Comment:
let's add comment/javadoc here describing what and where should be added when I want to add one more validator (e.g. in Validators.ts, but anywhere else?)

Reply:
I've updated PR description with detailed instruction of how to add new validator

Reply:
I would like to call it rather 'MandatoryValueValidator`  instead of `NonEmpty...`. It should be used in situation when something is not optional but mandatory. On the frontend we will interpret it by checking if field is non empty, but we should think about it in domain categories instead of presentation. This validators at the end should be interpreted on both FE and BE side.
Also we can introduce NonEmptyValidator for non empty strings (`''` in spel) or non empty lists (`{}` in spel) in the future.

Reply:
Ok,  done